tests/tests/assets.rs

```diff
@@ -0,0 +1,117 @@
+use fuel_core::{
+    chain_config::{
+        CoinConfig,
+        StateConfig,
+    },
+    database::Database,
+    service::{
+        Config,
+        FuelService,
+    },
+};
+use fuel_core_client::client::{
+    types::{
+        primitives::AssetId,
+        TransactionStatus,
+    },
+    FuelClient,
+};
+use fuel_core_types::fuel_tx::{
+    Input,
+    Output,
+    TransactionBuilder,
+    TxId,
+};
+
+async fn setup_service(configs: Vec<CoinConfig>) -> FuelService {
+    let state = StateConfig {
+        coins: configs,
+        ..Default::default()
+    };
+    let config = Config::local_node_with_state_config(state);
+
+    FuelService::from_database(Database::default(), config)
+        .await
+        .unwrap()
+}
+
+#[tokio::test]
+#[ignore] // TODO: Need to be able to mint assets for this test to work
+async fn asset_info() {
+    // setup test data in the node
+    let output_index = 5;
+    let tx_id = TxId::new([1u8; 32]);
+    let coin = CoinConfig {
+        output_index,
+        tx_id,
+        ..Default::default()
+    };
+
+    // setup server & client
+    let srv = setup_service(vec![coin]).await;
+    let client = FuelClient::from(srv.bound_address);
+
+    // run test
+    let asset_id = AssetId::new([1u8; 32]);
+    let mint_amount = 100;
+    let burn_amount = 50;
+
+    // Mint coins first
+    let mint_tx = TransactionBuilder::mint(
```

Comment:
I think this is not how you mint assets. From the spec : https://github.com/FuelLabs/fuel-specs/blob/master/src/tx-format/transaction.md#transactionmint this transaction is only used by the block producer that's why it's not accepted in the pool.
It's currently used for block rewards I think and it has this much fields for future usage.
However I don't know how to mint asset with the Native assets I never used it but I think it should be that way that you should mint your coins (using script or calls). Maybe someone else in the @FuelLabs/client has more details.

Reply:
Fixed it https://github.com/FuelLabs/fuel-core/pull/2445/commits/098b0317c87d1ed2712c15d84548e81129bf286e